frontend/test/metabase/scenarios/visualizations/scatter.cy.spec.js

```diff
@@ -68,16 +63,25 @@ describe("scenarios > visualizations > scatter", () => {
       },
     });
 
-    cy.wait("@dataset");
-
-    cy.get(".bubble")
-      .eq(13)
-      .trigger("mousemove");
-
+    triggerPopoverForBubble();
     popover().within(() => {
       cy.findByText("Created At:");
       cy.findByText("Orders count:");
       cy.findByText("Products count:");
     });
   });
 });
+
+function triggerPopoverForBubble(index = 13) {
+  cy.wait("@dataset");
+  // Hack that is needed because of the flakiness caused by adding throttle to the ExplicitSize component
+  // See: https://github.com/metabase/metabase/pull/15235
+  cy.get("[class*=ViewFooter]").within(() => {
```

Comment:
It might require adding more hacks like that in other tests. What do you think about a single hack like that https://github.com/metabase/metabase/pull/15522/files ?

Reply:
Not sure what is the right path to take. It feels weird hacking some JS code just to get tests to pass. The connection between the two might be lost/forgotten later, even with the nice and useful comment that you added.

@daltojohnso , @ariya - what is your take on this?